.gitignore

```diff
@@ -26,4 +26,7 @@ npm-debug.log
 # variables.
 /config/*.secret.exs
 
+# .env
```

Comment:
?

Reply:
basic_auth works with ENV_VARS so to run locally a .env file is needed.

Reply:
Right right, but this is `#.env`

You're including `.env` on L30

Reply:
the #.env is just a comment, I will edit it.